index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" type="text/css" href="./css/styles.css">
+  <link rel="stylesheet" type="text/css" href="./css/projects.css">
+  <link rel="stylesheet" type="text/css" href="./css/about.css">
+  <link rel="stylesheet" type="text/css" href="./css/form.css">
+  <title>Personal-Profolio</title>
+
+</head>
+
+<body>
+  <!-- Headline Section Starts Here -->
+
+  <header class="navigation">
+    <div class="logo">
+      <h2>My logo</h2>
+    </div>
+    <nav class="navlinks">
+      <img src="/Img/Hlogo.png">
+    </nav>
+  </header>
+  <section class="headline-main">
+    <div class="headline">
+      <h1 class="headline-heading heading-large">I'm Mariane <br> Glad to see you!</h1>
+      <p class="headline-paragraph paragraph">I’m a software developer! I can help you build a product , feature
+        or website
+        Look
+        through some of my work
+        and experience! If you like what you see and have a project you need coded, don’t hestiate to contact
+        me.
+      </p>
+      <p class="headline-sm-suport">LET’S CONNECT</p>
+      <ul class="social-icons">
+        <li><a href="#"><img src="/Img/-twitter.png"><i class="fa-brands fa-twitter"></i></a></li>
+        <li><a href="#"><img src="/Img/Linkedin icon.png"><i class="fa-brands fa-linkedin"></i></a></li>
+        <li><a href="#"><img src="/Img/-medium.png"><i class="fa-brands fa-medium"></i></a></li>
+        <li><a href="#"><img src="/Img/github.png"><i class="fa-brands fa-github"></i></a></li>
+        <li><a href="#"><img src="/Img/victory-icon.svg"><i class="fa-brands fa-facebook"></i></a></li>
```

Comment:
Kindly fix the issues with the image not displaying as explained earlier. so that our design looks exactly like the Figma design (e.g. font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). 🤗💪

Your design
![image](https://user-images.githubusercontent.com/78251927/235111156-c8482a4c-989d-4f3d-ad47-7758053b1274.png)

Figma
![image](https://user-images.githubusercontent.com/78251927/235111309-599e1dab-e1fa-4840-8297-b0672c78fd32.png)